src/main/kotlin/org/malachite/estella/process/api/RecruitmentProcessController.kt

```diff
@@ -40,7 +40,13 @@ class RecruitmentProcessController(
         @RequestBody stages: UpdateStagesListRequest
     ) =
         processService.updateStagesList(jwt, processId, stages.stages)
-            .let{ OwnResponses.SUCCESS }
+            .let { OwnResponses.SUCCESS }
+
+    @CrossOrigin
+    @GetMapping("/stages")
+    fun getAllStagesTypes() =
+        stageService.getAllStagesTypes()
+            .let { ResponseEntity.ok(it) }
```

Comment:
Czemu ten endpoint jest tutaj, skoro nie przekazujemy żadnych parametrów do funkcji getAllStagesTypes()?

Reply:
Nie wiem, gdzie indziej miałby być. Zwłaszcza że nie mamy żadnego kontrolera na stage Type, bo jest on niepotrzebny, to najbliżej byłoby do procesu